server/views/components/summary-card.njk

```diff
@@ -6,6 +6,11 @@
                 <h2 class="summary-card__title">
                     {{ params.titleHtml | safe if params.titleHtml else params.titleText }}
                 </h2>
+        {% if params.actions.items.length %}
```

Comment:
link should only show if person has been successfully booked into prison

Reply:
should this loop round each action?

Reply:
Opted for null check of `action.items` object as will only ever be one link within the summary card header. Refactored in [separate PR](https://github.com/ministryofjustice/hmpps-welcome-people-into-prison-ui/pull/378).